styx-api-service/src/main/java/com/spotify/styx/api/BackfillResource.java

```diff
@@ -152,20 +173,67 @@ private BackfillsPayload getBackfills(RequestContext requestContext) {
     }
   }
 
-  private Response<Void> haltBackfill(String id) {
+  private String schedulerApiUrl(CharSequence... parts) {
+    return schedulerServiceBaseUrl + SCHEDULER_BASE_PATH + "/" + String.join("/", parts);
+  }
+
+  private CompletionStage<Response<ByteString>> haltBackfill(String id, RequestContext rc) {
     try {
       final Optional<Backfill> backfillOptional = storage.backfill(id);
       if (backfillOptional.isPresent()) {
-        storage.storeBackfill(backfillOptional.get().builder().halted(true).build());
+        final Backfill backfill = backfillOptional.get();
+        storage.storeBackfill(backfill.builder().halted(true).build());
+        return haltActiveBackfillInstances(backfill, rc.requestScopedClient());
```

Comment:
We can notify use that some active instances cannot be halted, while no new ones will be triggered, in this case.